### PR TITLE
updated readme to include that OPR is in beta as pointed out by the i…

### PR DESCRIPTION
**Pull request by TerryTong-Git**
…ssues tab

Fixes #<45>

> It's a good idea to open an issue first for discussion.

- [ ] Tests pass
- [ ] Appropriate changes to README are included in PR

### CODE REVIEW

File: components/core/src/server/installable-endpoints/ingest.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { CustomRequestHandler } from "../customrequesthandler";
+import { IntegrationApi, OfferChange } from "../../coreapi";
+import { OprServer } from "../../../build/server/oprserver";
+
+
+export const ingestEndpoint = (api: IntegrationApi, s: OprServer): any => {
```
Comment (johndayrichter) [inline components/core/src/server/installable-endpoints/ingest.ts:6]: Please change the return type of this function to "unknown" (or, even better, the upcoming opr-core/JsonValue interface). Our lint rules disallow "any" types except when absolutely necessary.

Thanks!

File: README.md [hunk 2]
```diff
@@ -60,3 +60,7 @@ This library is the reference implementation for [the Open Product Recovery stan
 ### Start configuring your own OPR server
 
 Check out the `examples` folder for example working setups for various environments. A good place to start is the `local-starter` folder. The local starter example uses an sqlite database and includes some special endpoints to help generate fake offers so that its easier to see common settings for a server and how they work.
+
```
Comment (johndayrichter) [inline README.md:63]: Thanks for this! I think this change ought to be broken out into a different PR. We could merge that immediately.

**Review by johndayrichter — CHANGES_REQUESTED** [review]
Sorry this took me so long to review - we've been swamped with issue #79.

Thanks for fixing the README and adding those utility endpoints. As you'll see from the per-file comments, I think the README change ought to be broken out from the other change. PR #79 makes major changes to the way integrations are installed, which will force some other changes to the endpoints you've added. But I've still commented on those files, just to lay out some issues about package setup and everything.

Thanks so much for contributing, and I hope you'll consider breaking up this PR so we can take your README changes ASAP!

File: README.md [hunk 3]
```diff
@@ -60,3 +60,7 @@ This library is the reference implementation for [the Open Product Recovery stan
...
+### Version Warning
+
+The Current Version of OPR is in beta and the major version is 0. All changes to the library may by breaking changes. Please refer to the discussions for details on updates. (https://github.com/google/open-product-recovery/discussions)
```
Comment (rvenables) [inline README.md:66]: Note the updated github URL (https://github.com/CaravanStudios/open-product-recovery). @TerryTong-Git: Still interested in making this change?

Comment (rvenables) [reply README.md:66]: @TerryTong-Git thank you for your contribution. We're going to assume this is abandoned for now and make this change.

**Comment by rvenables** [comment]
Closing as abandoned

**Pull request closed without merging**